website/blog/2022-07-25-coreapp.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Streamlining app creation with React Native for Windows CoreApp"
+author: Alexander Sklar
+authorURL: https://twitter.com/alexsklar
+description: "Streamlining app creation with React Native for Windows CoreApp"
+is_blog: true
+published_date: July 25, 2022
+---
+
+React Native builds on top of the native platform for every OS it runs on; on Windows, this translates to producing a Universal Windows Platform (UWP) app. 
+
+We heard from some of our customers that build times for these types of apps can be pretty long - especially so when building the React Native for Windows framework from source. To mitigate this problem we offer pre-built binary distributions of the framework in the form of NuGet packages. This works pretty well but has a few shortcomings:
+
+1) Even when not building the framework, building a UWP requires several build steps behind the scenes (the steps below are what happens for C++/WinRT apps):
+    - the MIDL compiler is run to turn IDL files into Windows Metadata - this is required to produce information used by the XAML compiler
+    - the XAML compiler is invoked to convert XAML markup files into C++ code
+    - the MDMerge tool runs to merge all Windows Metadata files from the application
+    - C++/WinRT is invoked to turn Windows Metadata into C++ projection headers as well as component sources
+    - Only then, the C++ compiler and linker run on the app
+2) Native modules don't currently fully support consuming the RNW NuGet package that the app they are getting built into is using; there is work in progress to unlock this scenario. 
+3) There's no framework source to patch if you want to customize some behavior; this is usually not needed by the vast majority of customers, but listing this here for completeness.
+
+We set out to address the first item on the list above, since a lot of applications won't need to customize their native code; these _greenfield_ applications consist of the bare-bones native UWP app, and all of their app logic is confined to their JavaScript (and whatever native modules they require).
+
+## Enter `CoreApp`
```

Comment:
Recommend adding a "get started" section. Is this going to be the new default mode for apps created in 0.70?

Reply:
not default, I should clarify this is experimental / trying to get customer feedback from it.

Reply:
we don't currently have NuGet packages for canary so it'll be a little tricky for folks to build with NuGet. For the time being I will point folks to the coreapp sample app, and I'll update the blog post once this gets into a nuget (once 70 ships)